CMakeLists.txt

```diff
@@ -13,6 +13,8 @@ PROJECT(iroha
   VERSION 1.2.0
   LANGUAGES C CXX)
 
+find_package (Python3 COMPONENTS Interpreter Development)
```

Comment:
You may add it to dependancies.cmake.

And I cannot get why is it required to build Iroha? I've never met any coupled issues.